### DISCUSSION TITLE
How to use RPC interface between two backends?

### DISCUSSION THREAD

**Discussion by fatalstar** [iTwin/itwinjs-core#6017] [Q&A]
<img width="1106" alt="image" src="https://github.com/iTwin/itwinjs-core/assets/38130258/2e969ed5-4a79-4bdf-965f-dff797a3a14c">
As this picture, I want Backend B call on the RPC server of Backend C. 

I created a RPC client on Backend B, as the documention shows, 

BentleyCloudRpcManager.initializeClient(
      {
        info: {
          title: 'general-purpose-imodeljs-backend',
          version: 'v3.0',
        },
        uriPrefix: 'http://localhost:3001',
      },
      [AspectRpcInterface],
      RpcRoutingToken.default, // for token check
    );

And in Backend C, this RPC Interface has been registered.
But I get error in node_modules\@itwin\core-common\lib\cjs\rpc\core\RpcConfiguration.js:70，
instance = configurationConstructor[RpcRegistry_1.INSTANCE] = new configurationConstructor();
TypeError: configurationConstructor is not a constructor

itwin Version is 3.7.5

**Comment by pratikshan85** [comment]
are you saying you have deployed your own backend, call it backend X, that has the AspectRpcInterface and you would like to register that on your frontend? This way the frontend is now talking to backend Y (which has the standard RPC interfaces) and backend X at the same time? (want to make sure I understand the goal)

**Reply by fatalstar** [reply]
no, I want backend X ( as a client) call on backend Y,  as shown in picture Backend B could call Backend C. ( line in the lower right corner )
